devU-client/src/components/authenticatedRouter.tsx

```diff
@@ -45,8 +45,8 @@ const AuthenticatedRouter = () => (
     <Route exact path='/courses/:courseId/assignments/create' component={AssignmentCreatePage} />
     <Route exact path='/courses/:courseId/assignments/:assignmentId' component={AssignmentDetailPage} />
     <Route exact path='/courses/:courseId/assignments/:assignmentId/update' component={AssignmentUpdatePage} />
-    <Route exact path='/ncagtest' component={NonContainerAutoGraderForm} />
-    <Route exact path='/cagtest' component={ContainerAutoGraderForm} />
+    <Route exact path='/courses/:courseId/assignments/:assignmentId/ncag' component={NonContainerAutoGraderForm} />
```

Comment:
I see Keifer made this change too, I believe his idea is better createNCAG and createCAG.

Reply:
Not sure what this means

Reply:
just change the path to like `/courses/:courseId/assignments/:assignmentId/createNCAG` or /createCAG for all of the other link paths